_dashboards/visualize/maps-plugin.md

```diff
@@ -1,22 +1,35 @@
 ---
 layout: default
-title: Using the Maps plugin
-nav_order: 60
+title: Using maps 
+grand_parent: Building data visualizations
+parent: Using coordinate and region maps
+nav_order: 5
 redirect_from:
   - /dashboards/maps/
 ---
 
-# Using the Maps plugin
+# Using maps 
 
 With OpenSearch Dashboards, you can create maps to visualize your geographical data. OpenSearch lets you construct map visualizations with multiple layers, combining data across different indexes. You can build each layer from a different index pattern. Additionally, you can configure maps to show specific data at different zoom levels. OpenSearch maps are powered by the OpenSearch maps service, which uses vector tiles to render maps. 
 
-## Getting started
+## Creating a new map
 
-To get started, perform the following steps:
+You can create a new map from the **Maps** or **Visualize** workflows by performing the following steps: 
```

Comment:
Feel free to disregard this. I'm looking at the great screenshots you added. Are the "workflows" actual UI elements? If they're not items you can actually see labeled or interact with, I'm not sure we need to make them bold face. But I'm not in the UI. So move on if this comment isn't valid.

Reply:
Yes, they are actually menu items in the top menu... I don't know whether there is a better word than "workflow" here. Let's ask @natebower